compiler/src/iree/compiler/Tools/BUILD

```diff
@@ -182,9 +198,16 @@ cc_library(
     ],
     hdrs = ["iree_compile_lib.h"],
     deps = [
+        ":init_iree",
         "//compiler/src/iree/compiler/API2:Headers",
         "//compiler/src/iree/compiler/API2/Internal:Embed",
         "//compiler/src/iree/compiler/Pipelines",
         "@llvm-project//llvm:Support",
     ],
 )
+
+iree_compiler_cc_library(
```

Comment:
*sigh* it would be nice to actually fully support Bazel and add a build flag

Reply:
Closest I've gotten to getting this uniform/working is if I resort to both systems just generating files that get included here (genrule effectively). Or just use --defines in bazel which invalidates its cache.

Reply:
Well another alternative: the version is always set in the file and github action needs to modify the version file. Then its consistent across both and just a sed script in action. Then independent of cmake or bazel, its just a .sh script away.

Reply:
Got it working with build flags (quite cumbersome to read a string_flag ...) but it didn't provide any noticeable value above just defines & local_defines. Analysis was invalidated both times and recompilation was scoped both times, so went for simpler option.